src/applications/disability-benefits/all-claims/content/form0781/workflowChoicePage.jsx

```diff
@@ -0,0 +1,56 @@
+import { VaAdditionalInfo } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import React from 'react';
+
+export const workflowChoicePageTitle = (
+  <div className="vads-u-font-size--h3">
+    Statement about mental health conditions (VA Form 21-0781)
+  </div>
+);
+
+export const workflowChoicePageDescription = (
+  <>
+    <p>
+      You selected these mental health conditions: <b>PTSD, Depression</b>.
```

Comment:
This should display selections from [2.1] 0781 screener, `formData.mentalHealth.conditions`.

Reply:
Implemented in https://github.com/department-of-veterans-affairs/vets-website/pull/33655/commits/d68a885d16f38a3446cd4a5d5236936ecd7f76bc and https://github.com/department-of-veterans-affairs/vets-website/pull/33655/commits/6fea11d8dac18237e2e89033841b83d7dc47ef7d per the [latest copy revisions](https://github.com/department-of-veterans-affairs/va.gov-team/issues/101227)